src/components/ItemTable/ItemAvailability.test.tsx

```diff
@@ -57,11 +54,26 @@ describe("ItemAvailability", () => {
     render(<ItemAvailability item={item} />)
     expect(screen.getByText("Not available")).toBeInTheDocument()
     expect(screen.getByText("- Please", { exact: false })).toBeInTheDocument()
-    screen.getByRole("button", {
-      name: "contact a librarian",
-    })
+    expect(screen.getByText("contact a librarian")).toBeInTheDocument()
     expect(
       screen.getByText("for assistance.", { exact: false })
     ).toBeInTheDocument()
   })
+  it("pre-populates the metadata in the feedback form for unavailable items", async () => {
```

Comment:
is there a way to test the presence of the other item metadata? for business logic documentation more than anything else.

Reply:
will add this to the follow up testing ticket AC